pulsar-broker/src/main/java/org/apache/pulsar/broker/service/PrecisPublishLimiter.java

```diff
@@ -71,31 +80,63 @@ public void update(Policies policies, String clusterName) {
                 : null;
         this.update(maxPublishRate);
     }
+
     public void update(PublishRate maxPublishRate) {
-        if (maxPublishRate != null
```

Comment:
can we exit in case that previous `maxPublishRate` is the same as the current one ?
this way we will save resources

Reply:
I think that would make the logic a bit complicated without much gains.